server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -144,14 +145,22 @@ export function create(
 		async (request, response) => {
 			const tenantId = getParam(request.params, "tenantId");
 			const documentId = getParam(request.params, "id");
-			const signalContent = getParam(request.body, "singalContent");
+			const signalContent = request?.body?.signalContent;
+			if (!isValidSignalEnvelope(signalContent)) {
+				response
+					.status(400)
+					.send(`signalContent should contain 'content' and 'type' keys.`);
+				return;
+			}
 			try {
 				const signalRoom: IRoom = { tenantId, documentId };
 				const payload: IBroadcastSignalEventPayload = { signalRoom, signalContent };
 				collaborationSessionEventEmitter.emit("broadcastSignal", payload);
 				response.status(200).send("OK");
+				return;
```

Comment:
As we discussed offline, the `collaborationSessionEventEmitter.emit()` call a few lines above this needs a check for `undefined`.